Parameterize copy-resourcegroup-tag to maximize reusability.

rjygraham opened the pull request:
Instead of hardcoding the tag name as example, it can be specified as a parameter so that the Policy definition can be rolled up into a larger initiative encompassing multiple tags to be copied down to resources.

camillemarie commented on samples/ResourceGroup/copy-resourcegroup-tag/azurepolicy.json:
```diff
@@ -2,18 +2,26 @@
         "policyRule": {
             "if": {
-                "field": "tags.example",
+                "field": "[concat('tags.', parameters('tagName'))]",
```
>"field": "[concat('tags.', parameters('tagName'))]", [](start = 16, length = 52)

Should use bracket syntax to allow wider variety of parameter values (e.g. tag names with dashes in them)

`"field": "[concat('tags[', parameters('tagName'), ']')]",`

camillemarie commented on samples/ResourceGroup/copy-resourcegroup-tag/azurepolicy.parameters.json:
```diff
@@ -1 +1,8 @@
-{}
\ No newline at end of file
+{
```
README samples should be updated to account for the new parameter.

camillemarie commented:
Thanks! I've been meaning to do this, but hadn't gotten around to it yet.

rjygraham commented:
Thanks for the suggestions regarding bracket format and updating README. I've pushed out commit 06ab47707f8e5fb1349b3ade372a61a3ce86c123 which addresses those points.

camillemarie commented:
> $assignment = New-AzPolicyAssignment -Name <assignmentname> -Scope <scope> -PolicyDefinition $definition

Example should include parameter:

$assignment = New-AzPolicyAssignment -Name <assignmentname> -Scope <scope> -PolicyDefinition $definition *-tagName "costCenter"*

---
Refers to: samples/ResourceGroup/copy-resourcegroup-tag/README.md:19 in dcbb1ba. [](commit_id = dcbb1baf96038e1ad0059fa9da0161cd2222797a, deletion_comment = False)

camillemarie commented on samples/ResourceGroup/copy-resourcegroup-tag/README.md:
```diff
@@ -26,7 +26,7 @@ $assignment
+az policy definition create --name 'copy-resourcegroup-tag' --display-name 'Copy resource group tag to resource' --description 'Copy a tag specified in the parameter value from the resource group to the resource.  For example, a tag named costCenter on the resource group will be copied to the resources. Provide the actual tag name in tagName parameter at time of policy assignment.' --rules 'https ...
 
 # Create the Policy Assignment
 az policy assignment create --name <assignmentname> --scope <scope> --policy "copy-resourcegroup-tag" 
```
az cli example should include parameter:

--params "{'tagName':{'value': 'costCenter'}}"
